wsapi.go

```diff
@@ -399,9 +399,10 @@ func (s *Session) UpdateStatusComplex(usd UpdateStatusData) (err error) {
 }
 
 type requestGuildMembersData struct {
-	GuildID []string `json:"guild_id"`
-	Query   string   `json:"query"`
-	Limit   int      `json:"limit"`
+	GuildID  string   `json:"guild_id"`
+	GuildIDs []string `json:"guild_id"`
+	Query    string   `json:"query"`
+	Limit    int      `json:"limit"`
 }
```

Comment:
You can leave the struct the same as before so:
```go
type requestGuildMembersData struct {
	GuildID []string `json:"guild_id"`
	Query   string   `json:"query"`
	Limit   int      `json:"limit"`
}
```

And then pass the argument in the original function to the slice.